### PR TITLE
Multi network prep

### PR DESCRIPTION
**Pull request by samkohn**
This includes all of the commits from PR #9 (so tackle that first), plus some changes to make it easier to manage multiple neural net architectures in our repository. Specifically:
- Move data preprocessing from occurring in data_loaders (b4cdf77, #10) into the AbstractNetwork specification. This will allow us to load data the same way but preprocess differently, e.g. to scale to a standard deviation vs. an absolute min/max, by overriding the preprocess_data method.
- Implement my suggestion in #11 to ensure the train, test, and val sets are preprocessed in a consistent manner.
- Generalize the scale_min_max method to scale to arbitrary specified min and max (54e6b68).
- Add a README documentation file in the networks directory so we can consistently document the networks we create.

### CODE REVIEW

**Comment by eracah** [comment]
works well. I will add the learning rate thing. The loss changes very slowly, so I am thinking about increasing learning rate
